_bibliography/authorGuidelines.bib

```diff
@@ -0,0 +1,72 @@
+---
+---
+References
+==========
+@BOOK{ManualofStyle,
+title={The Chicago Manual of Style},
+PUBLISHER=" 17th ed. Chicago: University of Chicago Press",
+year="2017",
+url = {https://doi.org/10.7208/cmos17},
+}
+
+@BOOK{EditorialStyleManualAuthors,
+	title = {IEEE Editorial Style Manual for Authors},
+	language = {en},
+	PUBLISHER = {IEEE Publishing Operations},
+	year = "2023",
+	url = {https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE-Editorial-Style-Manual-for-Authors.pdf},
+}
+
+@BOOK{EditingMathematics,
+	title = {IEEE Editing Mathematics},
+	language = {en},
+	Publisher = {IEEE Publishing Operations},
+	year = "2023",
+	url = {https://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/Editing-Mathematics.pdf},
+}
+
+@ARTICLE{StandardDictionary,
+  author={},
+  journal={IEEE Transactions on Power Apparatus and Systems}, 
+  title={IEEE Standard Dictionary of Electrical and Electronics Terms}, 
+  year={1996},
+  volume={PAS-99},
+  number={6},
+  pages={37a-37a},
+  keywords={Dictionaries},
+  doi={10.1109/TPAS.1980.319816}}
+
+  @BOOK{AutomaticControlTerminology,
+  author={American National Standards Institute, American Society of Mechanical Engineers},
+  publisher={American Society of Mechanical Engineers}, 
+  title={American national standard supplement to terminology for automatic control ANSI C85.1-1963 : new and revised terms}, 
+  year={1973},
+  }
+
+    @BOOK{AutomaticControlSystemDictionary,
+  authors={ Rihard Karba and al.},
+  publisher={Springer Cham}, 
+  title={Terminological Dictionary of Automatic Control, Systems and Robotics}, 
+  year={2024},
+  doi={https://doi.org/10.1007/978-3-031-35755-8}
```

Comment:
Comma is missing
doi={https://doi.org/10.1007/978-3-031-35755-8},
While executing, encountered error

Reply:
corrections are provided in last push.